src/main/scala/reporter/RemoraDatadogReporter.scala

```diff
@@ -24,12 +23,13 @@ class RemoraDatadogReporter(metricRegistry: MetricRegistry, datadogConfig: DataD
     }
   }
 
-  private val metricNameFormatter: MetricNameFormatter = new MetricNameFormatter {
+  private def metricNameFormatter(remove_tags_from_metric_name: Boolean): MetricNameFormatter = new MetricNameFormatter {
```

Comment:
Same here, change not necessery - should be camel case.

Reply:
replaceAll FTW :/

Reply:
Yes, that was my guess... 😉